themes/default/content/docs/reference/yaml/_index.md

```diff
@@ -300,11 +301,32 @@ variables:
             values: ["amzn-ami-hvm-*-x86_64-ebs"]
         owners: ["137112412989"]
         mostRecent: true
+      Options:
+		Version: 5.9.0
       Return: id
 ```
 
 The expression `${AmazonLinuxAmi}` will return the AMI ID returned from the [`aws:getAmi`](https://www.pulumi.com/registry/packages/aws/api-docs/getami/) function.
 
+#### Invoke Options
+
+The value of the `options` property of an Fn::Invoke is an object with the following properties.
+
+The  `parent` and`provider` values permit expressions which must use interpolation syntax to reference resources by name. For example:
+
+```yaml
+    options:
+      provider: ${myEksProvider}
+      parent: ${someParentResource}
+```
+
+| Property                  | Type         | Description |
+| ------------------------- |--------------|-------------|
+| `parent`                  | Expression   | No | Yes | Parent specifies a parent for the resource |
+| `provider`                | Expression   | No | Yes | Provider specifies an explicitly configured provider, instead of using the default global provider |
```

Comment:
```suggestion
| `parent`                  | Expression   | Parent specifies a parent for the resource |
| `provider`                | Expression   | Provider specifies an explicitly configured provider, instead of using the default global provider |
```